modules/DMZ/data_kit/data_filler.py

```diff
@@ -2,35 +2,40 @@
 import numpy as np
 
 
-#Drops rows which have fewer than threshold observations.
-#e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
+# Drops rows which have fewer than threshold observations.
+# e.g. if threshold = 5, any rows with fewer than 5 observations will be removed from pandas_data.
```

Comment:
So I guess what extra documentation do you think we need? An example of calling the function with example data that shows how it is used? I want to document, but I also don't want to over-document.